whatwg.org/mailing-list

```diff
@@ -24,62 +24,102 @@
+  through GitHub</a>, but occasional announcements and discussion will also happen on
+  whatwg@whatwg.org.
 
-   <h2 id="specs">The specifications</h2>
+  <h3>Archives</h3>
 
-   <p>The WHAT Working Group discusses issues and new proposals on an
-   open and public mailing list, whatwg@whatwg.org. Discussion from
-   interested parties is welcome.</p>
+  <p>The <a href="http://lists.w3.org/Archives/Public/public-whatwg-archive/">mailing list
+  archives</a> are publicly accessible.</p>
 
-   <h3>Archives</h3>
+  <h3>Subscribing</h3>
 
-   <p>The <a href="http://lists.w3.org/Archives/Public/public-whatwg-archive/">mailing
-   list archives</a> are publicly accessible.</p>
+  <p>To subscribe to the mailing list, send an email to
+  <a href="mailto:whatwg-subscribe@whatwg.org">whatwg-subscribe@whatwg.org</a>, and then reply to
+  the confirmation email you receive. (Please disregard the suggestion that you follow links to a
+  web site; due to a limitation of our hosting provider, that site is currently unavailable.)
 
-   <h3>Subscribing</h3>
+  <h3>Posting</h3>
 
-   <p>To subscribe to the mailing list, send an email
-   to <a href="mailto:whatwg-subscribe@whatwg.org">whatwg-subscribe@whatwg.org</a>,
-   and then reply to the confirmation email you receive. (Please
-   disregard the suggestion that you follow links to a Web site; due
-   to a limitation of our hosting provider, that site is currently
-   unavailable.)
+  <p>To post a message to all the list members, first subscribe to the list as described above, then
+  send email to <a href="mailto:whatwg@whatwg.org">whatwg@whatwg.org</a>.</p>
 
-   <p>This is a high-volume list.</p>
+  <h3>Unsubscribing</h3>
 
-   <h3>Posting</h3>
+  <p>To unsubscribe from the mailing list, send an email to
+  <a href="mailto:whatwg-unsubscribe@whatwg.org">whatwg-unsubscribe@whatwg.org</a> from the address
+  you want to unsubscribe, and then reply to the confirmation email you receive.
 
-   <p>To post a message to all the list members, first subscribe to
-   the list as described above, then send email
-   to <a href="mailto:whatwg@whatwg.org">whatwg@whatwg.org</a>.</p>
+  <h3 id="moderation">Moderation</h2>
 
-   <h3>Unsubscribing</h3>
+  <p>The above list is not moderated. If you receive a message saying your message is in the
+  moderation queue, it means it has been caught by our spam filter. Make sure you subscribed and
+  posted using the right email address and make sure you didn't send any attachments. If you still
+  can't get your email sent to the list, please open a
+  <a href="https://github.com/whatwg/meta/issues/new">new meta issue</a>.</p>
 
-   <p>To unsubscribe from the mailing list, send an email
-   to <a href="mailto:whatwg-unsubscribe@whatwg.org">whatwg-unsubscribe@whatwg.org</a>
-   from the address you want to unsubscribe, and then reply to the
-   confirmation email you receive.
+  <h3>Can I use +1?</h3>
```

Comment:
I was wondering if we should copy this to the FAQ "how to interact on GitHub". Although, I think most people on GitHub have gotten the message to use the GitHub reactions feature instead of +1 messages, so maybe it's not necessary.

Reply:
I think it's worth making the advice explicit -- pose questions as suggested positions, then expect people to give an appropriate reaction.  For example, this question might be posed: "I think most people on GitHub have gotten the message to use the GitHub reactions feature instead of +1 messages, so it's not necessary.  Agree?"  Then I would have just given it a thumbs up rather than typing ;-)